network/containerizer/linkLayerDevicesForSpaces_test.go

```diff
@@ -0,0 +1,397 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package containerizer
```

Comment:
These tests have been moved from state and converted to mocks.  Please review to ensure unit test coverage has not changed.

Reply:
Can you rename the test file so it uses a snake-cased name?